pytest_django_test/settings_base.py

```diff
@@ -29,3 +29,5 @@
 ]
 
 DATABASE_ROUTERS = ['pytest_django_test.db_router.DbRouter']
+
+USE_TZ = False
```

Comment:
If we're setting it, probably better to set it to `True`, that's the common/useful value.

```suggestion
USE_TZ = True
```